.github/workflows/test-env-deploy.yml

```diff
@@ -3,6 +3,16 @@ name: TEST-ENV-DEPLOYMENT
 
 on: [pull_request]
 jobs:
+  check-secret:
+    runs-on: ubuntu-22.04
+    outputs:
+      testmo-url: ${{ steps.testmo-url.outputs.defined }}
+    steps:
+      - id: testmo-url
+        env:
+          TESTMO_URL: ${{ secrets.TESTMO_URL }}
+        if: "${{ env.TESTMO_URL != '' }}"
+        run: echo "::set-output name=defined::true"
```

Comment:
`::set-output` is deprecated https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/